### ISSUE TITLE
Add tests for prettyhtml cli

### ISSUE THREAD

**Issue by StarpTech** [Prettyhtml/prettyhtml#70]
As titled

**Comment by StarpTech** [comment]
@flurmbo Would you like to create on this issue?

**Comment by flurmbo** [comment]
Could you give some guidance on what sort of tests you're looking for? Making sure that the flags in prettyhtml/cli.js do what they're supposed to do for example

**Comment by StarpTech** [comment]
Exactly. You need to test the [cli](https://github.com/Prettyhtml/prettyhtml/blob/master/packages/prettyhtml/cli.js) and the prettyhtml [api](https://github.com/Prettyhtml/prettyhtml/blob/master/packages/prettyhtml/index.js).

**Comment by flurmbo** [comment]
Ok, I'll give it a try. I'll let you know when I've written a couple of tests so you can take a look and tell me if I'm doing it right

**Comment by StarpTech** [comment]
Sure :+1:
